test/action.spec.js

```diff
@@ -0,0 +1,112 @@
+const chai = require('chai');
+const expect = chai.expect;
+const sinon = require('sinon');
+const mock = require('mock-require');
+const log = require('npmlog');
+const path = require('path');
+
+const action = (config, args) => require('../src/action')(config, args || []);
+
+log.level = 'silent';
+
+describe('action', () => {
+
+  beforeEach(() => {
+    delete require.cache[require.resolve('../src/action')];
+  });
+
+  // that test suite has to be changed in next PR so we have
+  // fixed behaviour for `getProjectConfig` that's already in master
+  it('should return when run in a folder without package.json', () => {
+    const spy = sinon.spy(log, 'error');
+
+    const config = {
+      getProjectConfig: () => null,
+    };
+
+    action(config);
+
+    expect(spy.calledWith('ERRPACKAGEJSON')).to.be.true;
+  });
```

Comment:
That test suite tests older `rnpm-link` behaviour (where we used to return null). Has to be changed after being merged to master because another PR `sync-with-rnpm-changes` has been merged in the meantime. It's one-liner so I'll take care of that once you say `LGTM`.